Lesson 6 - Developer tools/homework/package.json

```diff
@@ -0,0 +1,25 @@
+{
+
+  "name": "homework",
+  "version": "1.0.0",
+  "description": "Homework for Lesson 6",
+  "main": "test.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/konradperzyna/Kurs_UL.git"
+  },
+  "author": "Konrad Perzyna",
+  "dependencies" : {
+    "gulp" : "3.x.x",
```

Comment:
It's better to use `^3.x.x` or `~3.2.2` semver modifiers - see more at https://docs.npmjs.com/misc/semver.